packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/index.js

```diff
@@ -23,7 +23,12 @@ import { unlock } from '../../private-apis';
 import { store as editSiteStore } from '../../store';
 
 const noop = () => {};
-const NAVIGATION_MENUS_QUERY = { per_page: -1, status: 'publish' };
+const NAVIGATION_MENUS_QUERY = {
+	per_page: 1,
+	status: 'publish',
+	order: 'desc',
+	orderby: 'date',
```

Comment:
These two arguments aren't required since [they are defaults](https://developer.wordpress.org/rest-api/reference/posts/#arguments), but I think it makes it easier to scan what query the component makes.